### PR TITLE
Fix divide by zero bug

### LINKED ISSUE #33: Panic on malformed input: attempt to divide by zero
With #32 applied which fixed the early panics fuzzer could further into the decoding code. Here are some samples discovered by AFL that trigger a panic: [divide-by-zero.zip](https://github.com/PistonDevelopers/image-tiff/files/2917296/divide-by-zero.zip)

Backtrace: 
```
thread 'main' panicked at 'attempt to divide by zero', /home/shnatsel/Code/image-tiff/src/decoder/mod.rs:539:12
stack backtrace:
   0: std::sys::unix::backtrace::tracing::imp::unwind_backtrace
             at src/libstd/sys/unix/backtrace/tracing/gcc_s.rs:49
   1: std::sys_common::backtrace::_print
             at src/libstd/sys_common/backtrace.rs:71
   2: std::panicking::default_hook::{{closure}}
             at src/libstd/sys_common/backtrace.rs:59
             at src/libstd/panicking.rs:211
   3: std::panicking::default_hook
             at src/libstd/panicking.rs:227
   4: std::panicking::rust_panic_with_hook
             at src/libstd/panicking.rs:491
   5: std::panicking::continue_panic_fmt
             at src/libstd/panicking.rs:398
   6: rust_begin_unwind
             at src/libstd/panicking.rs:325
   7: core::panicking::panic_fmt
             at src/libcore/panicking.rs:95
   8: core::panicking::panic
             at src/libcore/panicking.rs:59
   9: <tiff::decoder::Decoder<R>>::read_image
  10: std::panicking::try::do_call
  11: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:102
  12: afl::read_stdio_bytes
  13: std::rt::lang_start::{{closure}}
  14: std::panicking::try::do_call
             at src/libstd/rt.rs:59
             at src/libstd/panicking.rs:310
  15: __rust_maybe_catch_panic
             at src/libpanic_unwind/lib.rs:102
  16: std::rt::lang_start_internal
             at src/libstd/panicking.rs:289
             at src/libstd/panic.rs:398
             at src/libstd/rt.rs:58
  17: main
  18: __libc_start_main
  19: _start
```

Steps to reproduce are the same as in #28

### PR DESCRIPTION
**Pull request by birktj**
This fixes #33. Thanks again @Shnatsel! I assume we are still allowed to use your samples in our tests, if not please let me know.

### CODE REVIEW

**Comment by Shnatsel** [comment]
Yup, fine by me.

File: tests/fuzz_tests.rs [hunk 1]
```diff
@@ -40,3 +40,12 @@ fn inf_loop() {
         true
     });
 }
+
+// https://github.com/image-rs/image-tiff/issues/33
+#[test]
+fn divide_by_zero() {
+    test_directory("./tests/fuzz_images/divide_by_zero", |file| {
+        decode_tiff(file).ok();
```
Comment (HeroicKatora) [inline tests/fuzz_tests.rs]: This call to `Result::ok` doesn't seem to achieve anything.

Comment (birktj) [reply tests/fuzz_tests.rs]: Well it removes the `unused result` error, but other than that it doesn't have any real value.

Comment (HeroicKatora) [reply tests/fuzz_tests.rs]: The idiomatic way to express that is `let _ = decode_tiff(file);` in that case.

Comment (birktj) [reply tests/fuzz_tests.rs]: You are right, fixed this one and the others in the same file.

**Pull request merged**